templates/base.html

```diff
@@ -107,17 +107,24 @@ <h1><a href="{{ SITEURL }}">{{ SITETITLE }}</a></h1>
       {% if pages or LINKS %}
       <nav>
         <ul class="list">
+          {# Open links in new window depending on the LINKS_IN_NEW_TAB setting #}
+          {% macro link_target(link) -%}
+            {% if LINKS_IN_NEW_TAB not in ('no', 'none', false, 0, 'external') or (LINKS_IN_NEW_TAB == "external" and not link.startswith("/") and not link.startswith(SITEURL)) %}
```

Comment:
more verbose is more readable in this case
```jinja2
{% if LINKS_IN_NEW_TAB == 'all' %}
{% else %}
  {% if LINKS_IN_NEW_TAB == 'external' %}
   ....
  {% else %}
  ....

Or similar
  {% endif %}
{% endif %}
```

Reply:
This is how it would look like, but honestly I don't like it. I'm keeping backwards compatibility, so that includes checking whether `LINKS_IN_NEW_TAB` is defined. That's why I wrote the condition the way I did, it includes that case.

```
{% if LINKS_IN_NEW_TAB is defined %}
  {% if LINKS_IN_NEW_TAB in ('all', 'yes', true, 1) %}
    target="_blank"
  {% endif %}
  {% if LINKS_IN_NEW_TAB == "external" and not link.startswith("/") and not link.startswith(SITEURL) %}
    target="_blank"
  {% endif %}
{% else %}
  target="_blank"
{% endif %}
```

Reply:
I see, well ok.